sklearn/ensemble/_iforest.py

```diff
@@ -463,6 +465,27 @@ def _more_tags(self):
         }
 
 
+_average_path_length_small = np.array((
```

Comment:
could you add a small comment above to say that this is a lookup table used in the `_average_path_length` function below?

Reply:
done